crates/bevy_ecs/src/world/mod.rs

```diff
@@ -2332,7 +2354,7 @@ impl World {
     ///
     /// This function will panic if any of the associated entities do not exist.
     ///
-    /// For the non-panicking version, see [`World::try_insert_batch`].
+    /// For the fallible version, see [`World::try_insert_batch`].
     #[track_caller]
     pub fn insert_batch<I, B>(&mut self, batch: I)
```

Comment:
I would prefer to merge all of the fallible / infallible variants, but that should be done in follow-up.